src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.bigBox {
+  position: relative;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.bigIcon {
+  border: none;
+  background-color: #fff;
+  top: 25px;
+  left: 26px;
+  position: absolute;
+  color: #687c94;
+  height: 19px;
+  width: 19px;
+}
+
+.searchBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: bold;
+  color: #3d4e61;
+}
+
+.searchBar::placeholder {
+  font-family: Avenir, Arial, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  color: #3d4e61;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar:focus {
+  box-shadow: none;
+  border-color: 1px solid #e1e7ed;
+  outline: none;
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+}
+
+.big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
```

Comment:
put these styles inside searchBar component, and override some of them in searchBar--small